src/components/datepicker/dateLocaleProvider.js

```diff
@@ -158,6 +158,10 @@
         if (!date) {
           return '';
         }
+				
+        if (typeof date === 'string') {
```

Comment:
When the date comes from the database, comes in string format. I also realized that require further adjustment, because it ends up disabling the calendar so we need to convert the $ scope before reaching the DatePicker.